plugin/pkg/auth/authorizer/rbac/bootstrappolicy/testdata/cluster-roles.yaml

```diff
@@ -863,6 +884,60 @@ items:
     - get
     - list
     - watch
+  - apiGroups:
+    - resource.k8s.io
+    resources:
+    - deviceclasses
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - resource.k8s.io
+    resources:
+    - resourceclaims
+    verbs:
+    - get
+    - list
+    - patch
+    - update
+    - watch
+  - apiGroups:
+    - resource.k8s.io
+    resources:
+    - resourceclaims/status
+    verbs:
+    - get
+    - list
+    - patch
+    - update
+    - watch
+  - apiGroups:
+    - ""
+    resources:
+    - pods/finalizers
+    verbs:
+    - get
+    - list
+    - patch
+    - update
+    - watch
```

Comment:
pre-existing, but only the "update" verb is meaningful for this, might as well prune the verb set to just "update" when building the kubeSchedulerRules for this

Reply:
Fixed with a cleanup commit "DRA RBAC: fix kube-scheduler bootstrap policy", then updating the following "DRA: graduate DynamicResourceAllocation feature to GA" commit accordingly.